Copy to SD Card root directory to update/config.ini

```diff
@@ -461,8 +461,8 @@ ext_speed:S60 N600 F1200
 #   Options: [disable: 0, enable: 1]
 auto_load_leveling:1
 
-#### Onboard / Printer SD Card Support
-# On Marlin firmware, the TFT will auto-detect Onboard SD Card.
+#### Onboard / Printer Media Support
```

Comment:
the same changes should be applied to the other two config.ini in the project

Reply:
Roger that!